indy_node/test/request_handlers/conftest.py

```diff
@@ -36,7 +42,7 @@ def schema_request():
     return Request(identifier=randomString(),
                    reqId=5,
                    signature="sig",
-                   operation={'type': SCHEMA,
+                   operation={'type': "101",
```

Comment:
Why did you change constant's using to string? If SCHEMA constant will be change by some reason then it will to hard to change all of strings "101" to another instead of changing constant's value.

Reply:
changing SCHEMA constant is an API breaking change. I think the tests should catch that. I think the benefit of catching accidental API changes in testing is worth the effort required to update test during an API change. 
I can change that back if it goes against the project design.